### PR TITLE
IA-2429 Initial Commit

### PR DESCRIPTION
**Pull request by mathvdh**
Preparation pour formation burkina

Related JIRA tickets : IA-2429

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [x] My migrations file are included
- [ ] Are there enough tests

### CODE REVIEW

File: iaso/api/stocks.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    def create(self, validated_data):
+        profile = self.context["request"].user.iaso_profile
+        wfv = StockItem.objects.create(account=profile.account, **validated_data)
+        return wfv
+
+
+class StockItemViewSet(ModelViewSet):
+    f"""Stock Items API
+
+    This API is restricted to authenticated authenticated users having the permission "{permission.STOCKS}"
```
Comment (madewulf) [inline iaso/api/stocks.py]: Bi authenticated ?

File: iaso/api/stocks.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    def get_queryset(self):
+        profile = self.request.user.iaso_profile
+        qs = StockItem.objects.filter(account=profile.account).order_by("id")
+        return qs
+
+    def destroy(self, request, pk=None):
+        profile = request.user.iaso_profile
+        existing = StockItem.objects.get(id=pk)
+        if existing.account != profile.account:
+            raise ValueError("You can only delete stock items from your own account")
```
Comment (madewulf) [inline iaso/api/stocks.py]: To me this should not be needed thantks to get_queryset that provide a queryset that filters on the possible StockItems

File: iaso/api/stocks.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+
+class StockMovementSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = StockMovement
+        fields = ["id", "stock_item", "org_unit", "quantity", "creation_date"]
+
+    def create(self, validated_data):
+        profile = self.context["request"].user.iaso_profile
+        wfv = StockMovement.objects.create(account=profile.account, **validated_data)
```
Comment (madewulf) [inline iaso/api/stocks.py]: what's wfv?

Comment (mathvdh) [reply iaso/api/stocks.py]: It's from a copy/paste and I forgot to change

File: iaso/api/stocks.py (hunk 2 above)
Comment (mathvdh) [reply iaso/api/stocks.py]: Removed it and will test it

**Pull request closed without merging**